stores/moderationSubmissionsStore.ts

```diff
@@ -0,0 +1,55 @@
+import { gql } from "graphql-request";
+import { defineStore } from "pinia";
+import { Locale, Submission } from "~/typedefs/gqlTypes.js";
+import { gqlClient } from "../utils/graphql.js";
+import { Ref, ref } from "vue";
+
+export const useModerationSubmissionsStore = defineStore(
+    "submissionsStore",
+    () => {
+        const submissionsData: Ref<Submission[]> = ref([]);
+
+        async function getSubmissions() {
+            const submissionsSearchResults = await querySubmissions();
+            submissionsData.value = submissionsSearchResults;
+        }
+
+        return { getSubmissions, submissionsData };
+    }
+);
+
+async function querySubmissions() {
+    try {
+        const submissionsFilters = {
+            filters: {
+                id: undefined,
+            },
+        };
+
+        const result = await gqlClient.request<{ submissions: Submission[] }>(
+            getSubmissions,
```

Comment:
rename to improve readability
```suggestion
            getSubmissionsGqlQuery,
```

Reply:
I will do this. I will also refactor the gql to get all the data if that is ok?